spark/dl/src/test/scala/com/intel/analytics/bigdl/dataset/SampleSpec.scala

```diff
@@ -61,6 +62,32 @@ class SampleSpec extends FlatSpec with Matchers {
       Some(featureParam), Some(labelParam)).set(samples)
   }
 
+  "create Sample" should "work fine" in {
+    val st1 = Tensor.sparse(Tensor.range(1, 10, 1))
+    val st2 = Tensor.sparse(Tensor.range(1, 10, 1))
+    val dt1 = Tensor.range(1, 10, 1)
+    val dt2 = Tensor.range(1, 10, 1)
+    val label1 = Tensor(1).fill(1)
+    val label2 = Tensor(1).fill(2)
+
+    Sample(st1)
```

Comment:
does this cover all sample factory methods?

Reply:
One missing

Reply:
Added